src/lib/pages/backoffice/Item/ItemDetails/ItemHeader.js

```diff
@@ -8,9 +8,45 @@ import { BackOfficeRoutes } from '@routes/urls';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import { Header } from 'semantic-ui-react';
+import { Header, Label } from 'semantic-ui-react';
+import { invenioConfig } from '@config';
 
 export class ItemHeader extends Component {
+  statusLabel = () => {
+    const { data } = this.props;
+    let status = {};
```

Comment:
Since its is expected to have a `value` and a `text` properties, may would make sense to initialize them here.